src/style.css

```diff
@@ -0,0 +1,66 @@
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-style: normal;
+  font-weight: 800;
+}
+
+body {
+  font-family: Avenir, sans-serif;
+  font-weight: 300;
+}
+
+.form {
+  margin-top: 20px;
+}
+
+.search {
+  width: 100%;
+  font-family: Avenir, sans-serif;
+  font-size: 16px;
+  font-weight: 800;
+  color: #3d4e61;
+  text-shadow: 0 4px 4px rgba(0, 0, 0, 0.25);
+  background: url(images/Search.svg) 26px center no-repeat;
+  padding: 23px 10px 23px 62px;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  outline: none;
+  box-sizing: border-box;
```

Comment:
Use `box-sizing` for the `*` selector: https://www.w3schools.com/cssref/sel_all.php
```
* {
  box-sizing: border-box;
}
```